libs/wingsdk/src/target-tf-azure/bucket.ts

```diff
@@ -77,7 +77,7 @@ export class Bucket extends cloud.Bucket {
     this.storageContainer = new StorageContainer(this, "Bucket", {
       name: storageContainerName,
       storageAccountName: this.storageAccount.name,
-      containerAccessType: this.public ? "public" : "private",
+      containerAccessType: this.public ? "blob" : "private",
```

Comment:
Just want to make sure I understand. `blob` allows access to the blobs in the container but unlike `public` does not give access to metadata and properties on container?

Reply:
There are no `public` unfortunately- https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_container.html#container_access_type
do you think `container` will be better than blob?